### PR TITLE
Catch UnsupportedOperationException from toJson

### PR DESCRIPTION
**Pull request by dmikurube**
For a better compatibility, just in case...

### CODE REVIEW

File: src/main/java/org/embulk/util/config/Compat.java [hunk 1]
```diff
@@ -105,6 +105,10 @@ private static Optional<String> callToJsonIfAvailable(final DataSource source) {
             jsonStringObject = toJson.invoke(source);
         } catch (final InvocationTargetException ex) {
             final Throwable targetException = ex.getTargetException();
+            if (targetException instanceof UnsupportedOperationException) {
+                // If the plugin's embulk-util-config does not implement toJson, it cannot retrieve a stringified JSON.
```
Comment (hiroyuki-sato) [inline src/main/java/org/embulk/util/config/Compat.java:109]: What do you think to add logging message?

**Review by hiroyuki-sato — APPROVED** [review]
LGTM👍

File: src/main/java/org/embulk/util/config/Compat.java (hunk 1 above)
Comment (dmikurube) [reply src/main/java/org/embulk/util/config/Compat.java:109]: This part is still an "expected" path, then I'm afraid that logging here could be noisy.

Comment (hiroyuki-sato) [reply src/main/java/org/embulk/util/config/Compat.java:109]: Thanks! LGTM👍

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**